server/mongodb/models/Nonprofit.ts

```diff
@@ -11,6 +11,10 @@ const NonprofitSchema = new Schema<Nonprofit>({
     type: AddressSchema,
     required: true,
   },
+  contact: {
```

Comment:
Ok this appears to be a misinterpretation of the design, which is my fault because I should have explained it in the issue.  The contact is actually a user, so this should be an ObjectId that references the user model.  The dropdown in the design is meant to be a way for a nonprofit to select their representative or main point of contact from all of their users.  This is honestly an easy fix to make throughout the implementation that I will handle because I did not make it clear to you.